secboot/keygen.sh

```diff
@@ -8,13 +8,15 @@ else
     ROOT="$(cd -- "${SCRIPT_DIR}/.." && pwd)"
 fi
 
+TARGET="${1:-}"
+
 CONF="${CONF:-./conf}"
```

Comment:
For consistency with how OUT_DIR is constructed, the latter part should be "${PWD}/conf". I assume we don't want the directory to be relative to whatever directory the script 'cd' to (there's no such directory changes at the moment, though)